micro_sam/_vendored.py

```diff
@@ -59,3 +62,31 @@ def batched_mask_to_box(masks: torch.Tensor) -> torch.Tensor:
         out = out[0]
 
     return out
+
+
+# segment_anything/util/amg.py
+# https://github.com/facebookresearch/segment-anything
```

Comment:
We should really say more about:
1. Why we felt we needed to put these functions here, and 
2. What condition needs to be met to remove it and go back to a more official implementation. How will someone know in future when it is possible or desirable to remove these workarounds?

And yes, I didn't write either of those things down when I added a function here 😆 Oops, I think that was a mistake and we should fix that now too.

Reply:
I didn't see that before merging; I will go ahead and fix it up on `dev` directly ;).